app/models/form/lettings/questions/stock_owner.rb

```diff
@@ -11,8 +11,14 @@ def initialize(id, hsh, page)
 
   def answer_options
     answer_opts = { "" => "Select an option" }
+
     return answer_opts unless ActiveRecord::Base.connected?
     return answer_opts unless current_user
+    return answer_opts unless log
+
+    if log.owning_organisation_id.present?
+      answer_opts = answer_opts.merge({ log.owning_organisation.id => log.owning_organisation.name })
+    end
```

Comment:
Whatever is currently in the database, we want to make sure is in the list of answer options.

Reply:
although not introduced by this PR. line 2 `attr_accessor :current_user, :log` should be removed and those passed into the method calls. otherwise this singleton will suffer from side effects. would it be an easy change for line 12 from
```
def answer_options
```
to
```
def answer_options(log, current_user)
```
or something to that effect?

Reply:
to be addressed later